src/logging/request-obfuscator.js

```diff
@@ -49,17 +49,19 @@ function obfuscateResponse(request, instance) {
     return;
   }
 
+  request.body = JSON.parse(request.body);
```

Comment:
@pedrobranco shouldn't we be keeping the same string as before?

Reply:
Fixed.

Reply:
I suggest passing the parsed body as an argument instead of mutating `request.body` here. I would then stringify the resulting (mutated) `body` object at the end of the method.

Reply:
Oh sure.